script/Arcueid/Day 6/06_00_ARC06_4.txt

```diff
@@ -398,12 +399,12 @@ is the figure of the woman who no longer needs me,
 [sha:6ccab5357ac16a5b2150f4a1fd8605c31cd6883a]{
 -- Page 350, Offset 14786.
 -- 「ああもう、今日は止め止め！」
-"Ah man, today's not my day!"
+"Ah man, I've had enough of today!"
```

Comment:
The english equivalent of what he is saying here would be "Aauh, I'm calling it a day ". But may differ more from the japanese...